Flick/Controllers/ListViewController.swift

```diff
@@ -70,18 +75,28 @@ class ListViewController: UIViewController {
         mediaCollectionView.bounces = false
         view.addSubview(mediaCollectionView)
 
-        if list.shows.count == 0 {
-            setupEmptyStateViews()
-        }
-
         setupSections()
         setupConstraints()
     }
 
-    init(list: MediaList) {
-        super.init(nibName: nil, bundle: nil)
-        self.list = list
-        self.listSummaryHeight = list.tags.isEmpty ? 80 : 145
+    override func viewDidAppear(_ animated: Bool) {
```

Comment:
do we do network call in viewdidappear or viewwillappear? ive been using viewwillappear

Reply:
I think it might be better to put it in `viewDidAppear` because the view controller is pushed instantly. If we put in `viewWillAppear` and the networking call is slow, then the view controller doesn't get pushed until the call to complete. I think the user might be confused why the VC is not pushed, so it could be poor UX. Also putting it in `viewDidAppear` is good for if we want to add in loading animations